src/lwc/utilIllustration/utilIllustration.html

```diff
@@ -1049,10 +1049,26 @@
                 class="slds-text-heading_medium">
                 {title}
             </h3>
-            <p if:true={hasMessage}
-               class="slds-text-body_regular">
-               {message}
-            </p>
+            <template if:true={shouldWrapText}>
+                <lightning-layout>
+                    <lightning-layout-item size="12"
+                                           large-device-size="6"
+                                           small-device-size="6"
```

Comment:
I think it'd be great to let the layout fill and use all the available real estate for small devices.

```suggestion
                                           small-device-size="12"
```